prow/plugins/plugins.go

```diff
@@ -439,6 +440,13 @@ type MergeWarning struct {
 	BranchWhiteList map[string][]string `json:"branch_whitelist,omitempty"`
 }
 
+// Welcome is config for the welcome plugin
+type Welcome struct {
+	// Message is the welcome message to post on new-contributor PRs
+	// TODO(bentheelder): make this be configurable per-repo?
```

Comment:
I think I want to tackle that in a follow-up, once we verify that the "first PR detection" works against a trial repo (test-infra?). Probably with a default message in the config and then per-repo overrides?

Reply:
Oh yes, sorry I was just 👍 the idea of making it configurable at some point. Maybe I could try to add that after you've done all the hard work of plumbing in the plugin. 😀

Reply:
Ok great, I think it will be fairly small but we're already at size/L :grimacing: 
We also need to come up with great welcome message(s) to put in the config next :-)

Reply:
I think maybe it should even operate as a go template over an info struct with the repo, org etc.. just doing that might be enough

Reply:
neat idea!

Reply:
I implemented this